packages/freerdp.rb

```diff
@@ -0,0 +1,34 @@
+require 'package'
+
+class Freerdp < Package
+  description 'FreeRDP is a free implementation of the Remote Desktop Protocol.'
+  homepage 'http://www.freerdp.com/'
+  version '2.0.0-rc4'
+  source_url 'https://github.com/FreeRDP/FreeRDP/archive/2.0.0-rc4.tar.gz'
+  source_sha256 '3406f3bfab63f81c1533029a5bf73949ff60f22f6e155c5a08005b8b8afe6d49'
+
+  depends_on 'ffmpeg'
+  depends_on 'gst_plugins_base'
+  depends_on 'cups'
+  depends_on 'xmlto'
+  depends_on 'libxslt'
+  depends_on 'ninja'
+  depends_on 'sommelier'
+
+  def self.build
+    system 'cmake', "-DCMAKE_INSTALL_PREFIX:PATH=#{CREW_PREFIX}", '-DWITH_SERVER=yes', '-DWITH_CLIENT=yes', '-DWITH_CHANNELS=yes',
+      '-DBUILD_SHARED_LIBS=yes', '-DWITH_GSSAPI=yes', '-DWITH_DSP_FFMPEG=yes', '-DWITH_LIBSYSTEMD=OFF', '-DWITH_CUPS=ON',
+      '-DWITH_PULSE=ON', "-DDOCBOOKXSL_DIR=#{CREW_PREFIX}/share/doc/docbook-xsl-1.79.2", '-GUnix Makefiles', '.'
+    system 'make'
+  end
+
+  def self.install
+    system 'make', 'install', "DESTDIR=#{CREW_DEST_DIR}"
+  end
+
+  def self.postinstall
+    puts 'To test this package, try the following command to connect to the Windows server at https://www.secure-od.com/:'.lightblue
```

Comment:
Please add empty `puts` lines at the top and bottom of the postinstall. It just looks better.